src/mol-gl/shader/postprocessing.frag.ts

```diff
@@ -70,6 +79,7 @@ float getOutline(const in vec2 coords, const in float opaqueDepth, out float clo
             if (sampleOutline == 0.0 && sampleOutlineDepth < closestTexel) {
                 outline = 0.0;
                 closestTexel = sampleOutlineDepth;
+                isTransparent = sampleOutlineCombined.a;
```

Comment:
is that a general fix for which you can do a separate PR?

Reply:
This is just useful with the new postprocessing pass configuration. Since opaque and transparent color blending is done within the postprocessing fragment it is now possible to apply opaque and transparent outlines separately similarly to ssao